authorization_services/topics/policy-client-policy.adoc

```diff
@@ -3,7 +3,12 @@
 
 You can use this type of policy to define conditions for your permissions where a set of one or more clients is permitted to access an object.
 
+ifeval::[{project_community}==true]
+To create a new client-based policy, select *Client* from the policy type list.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new client-based policy, select *Client* in the item list in the upper right corner of the policy listing.
+endif::[]
 
 .Add a Client Policy
 image:images/policy/create-client.png[alt="Add a Client Policy"]
```

Comment:
This is the old image.

Reply:
Also, there is an old image in policy-client-scope-policy-required-client-scope.adoc